conu/apidefs/image.py

```diff
@@ -162,6 +164,26 @@ def create_container(self, container_params):
         """
         raise NotImplementedError("create_container method is not implemented")
 
+
+class ImageCleanupPolicy(enum.Enum):
```

Comment:
I think this should be placed in apidefs/backend and named either CleanupPolicy or BackendCleanupPolicy. This name and location just doesn't make sense to me.